syncplay/messages_fi.py

```diff
@@ -0,0 +1,542 @@
+# coding:utf8
+
+"""Finnish dictionary"""
+
+# Filename, dictionary name and LANGUAGE-TAG value based on ISO country code. Language tag listed at https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c?redirectedfrom=MSDN
+
+fi = {
+     "LANGUAGE": "Finnish",
+     "LANGUAGE-TAG": "fi",
+
+     # Strings for Windows NSIS installer
+     "installer-language-file": "Finnish.nlf", # Relevant .nlf file at https://github.com/kichik/nsis/tree/master/Contrib/Language%20files
+     "installer-associate": "Yhdistä Syncplay toimimaan multimedia-tiedostojen kanssa.",
+     "installer-shortcut": "Luo oikopolut seuraaviin kohteisiin:",
+     "installer-start-menu": "Käynnistysvalikko",
+     "installer-desktop": "Työpöytä",
+     "installer-quick-launch-bar": "Pikakäynnistyspalkki",
+     "installer-automatic-updates": "Tarkista päivitykset automaattisesti",
+     "installer-uninstall-configuration": "Poista asetustiedosto.",
+
+     # Client notifications
+     "config-cleared-notification": "Asetukset pyyhitty. Muutokset tallennetaan kun toimitat uuden kelvollisen asetuskokonaisuuden.",
+
+     "relative-config-notification": "Asiaankuuluva asetustiedosto(t) ladattu: {}",
+
+     "connection-attempt-notification": "Yritetään yhdistää kohteeseen {}:{}",  # Port, IP
+     "reconnection-attempt-notification": "Yhteys palvelimeen menetettiin, koetetaan uudelleenyhdistämistä",
+     "disconnection-notification": "Yhteys palvelimeen katkaistu",
+     "connection-failed-notification": "Yhteydenotto palvelimeen epäonnistui",
+     "connected-successful-notification": "Yhteys onnistuneesti muodostettu palvelimeen",
+     "retrying-notification": "%s, Uusi yritys %d sekunnissa...",  # Seconds
+     "reachout-successful-notification": "Tavoitettu onnistuneesti {} ({})",
+
+     "rewind-notification": "Taaksepäin kelattu koska ilmeni aikaero {}",  # User
+     "fastforward-notification": "Pikakelaus eteenpäin koska ilmeni aikaero {}",  # User
```

Comment:
using the word "koska" ("because") requires a comma and I think these would be better expressed:

```suggestion
     "rewind-notification": "Kelattiin taaksepäin aikaeron {} vuoksi",  # User
     "fastforward-notification": "Pikakelattiin eteenpäin aikaeron {} vuoksi",  # User
```

Reply:
So go ahead? I think this translation should be gotten in and into polishing it all the time, then.

Reply:
#563